dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessServiceImpl.java

```diff
@@ -2989,7 +2988,7 @@ public TaskInstance releaseTaskGroup(TaskInstance taskInstance) {
             return null;
         }
         try {
-            while (taskGroupMapper.releaseTaskGroupResource(taskGroup.getId(), taskGroup.getUseSize()
+            while (thisTaskGroupQueue.getForceStart() != 1 && taskGroupMapper.releaseTaskGroupResource(taskGroup.getId(), taskGroup.getUseSize()
```

Comment:
```suggestion
            while (thisTaskGroupQueue.getForceStart() == Flag.NO && taskGroupMapper.releaseTaskGroupResource(taskGroup.getId(), taskGroup.getUseSize()
```

Reply:
ok thanks for the review suggestion, i have changed